docs/platform-specification.md

```diff
@@ -5,31 +5,62 @@ Platforms add support for new boards to the Arduino development software. They a
 [Boards Manager](package_index_json-specification.md) or manual installation to the _hardware_ folder of Arduino's
 sketchbook folder (AKA "user directory").<br> A platform may consist of as little as a single configuration file.
 
-## Hardware Folders structure
+## Platform installation directories
 
-The new hardware folders have a hierarchical structure organized in two levels:
+If the platforms are installed using the Board Manager the installation directory location will be as follow:
 
-- the first level is the vendor/maintainer
-- the second level is the supported architecture
+`{directories.data}/packages/{VENDOR_NAME}/hardware/{ARCHITECTURE}/{VERSION}/...`
 
-A vendor/maintainer can have multiple supported architectures. For example, below we have three hardware vendors called
-"arduino", "yyyyy" and "xxxxx":
+- `{directories.data}` is the data directory as specified in the
+  [configuration file](configuration.md#default-directories).
+- `{VENDOR_NAME}` is the identifier of the vendor/maintainer of the platform.
+- `{ARCHITECTURE}` is the architecture of the CPU used in the platform.
+- `{VERSION}` is the platform version.
 
+Alternatively, a platform may be manually installed by the user inside the Sketchbook/user directory as follows:
+
+`{directories.user}/hardware/{VENDOR_NAME}/{ARCHITECTURE}/...`
+
+- `{directories.user}` is the user directory as specified in the
+  [configuration file](configuration.md#default-directories).
+- `{VENDOR_NAME}` is the identifier of the vendor/maintainer of the platform.
+- `{ARCHITECTURE}` is the architecture of the CPU used in the platform.
+
+A vendor/maintainer can have multiple supported architectures.
+
+For example, below we have four platforms downloaded from three hypothetical hardware vendors called "arduino", "foo"
+and "bar", and installed using the Board Manager:
+
+```
+{directories.data}/packages/arduino/hardware/avr/1.2.3/...
+{directories.data}/packages/arduino/hardware/sam/1.4.5/...
+{directories.data}/packages/foo/hardware/avr/1.0.2/...
```

Comment:
Would it make more sense to use something else than "avr" here (and below)? It looks strange when you have three hardware vendors, but all three manufacture the same "architecture" avr, as avr is specific only to a specific vendor.

Use some other vendor/architecture combination instead for the foo and bar examples.

Reply:
I've updated the PR with a real example (with more architecture/vendors combination)